### PR TITLE
better filtering of player page wins table

### PR DESCRIPTION
**Pull request by Kramin42**

### CODE REVIEW

File: scoreboard/webutils.py [hunk 1]
```diff
@@ -155,22 +155,41 @@ def format_trow(game: orm.Game, index: int,
...
+                        }},
+                        "pagingType": "numbers"
+                    }});
+                    
+                    $('#{id}_wrapper input[type=search]')
+                        .off('cut input keypress keyup paste search')
+                        .on( 'keyup change', function () {{
+                        //console.log("searching for: "+this.value);
+                        var tokens = this.value.trim().split(/\s+/);
+                        // modify tokens to be \bXXXX, if two letters also let match \b..XX\b
```
Comment (alexjurkiewicz) [inline scoreboard/webutils.py]: could you add a comment explaining _why_ we do this?

File: scoreboard/webutils.py [hunk 2]
```diff
@@ -155,22 +155,41 @@ def format_trow(game: orm.Game, index: int,
...
+                        tokens = tokens.map(function(t) {{
+                            t = "\\\\b" + t + (t.length==2 ? "|\\\\b.."+t+"\\\\b" : "");
+                            return t;
+                        }});
+                        //console.log(tokens);
```
Comment (alexjurkiewicz) [inline scoreboard/webutils.py]: remove any commented out debug lines

File: scoreboard/webutils.py [hunk 3]
```diff
@@ -155,22 +155,41 @@ def format_trow(game: orm.Game, index: int,
...
+                        // AND tokens together
+                        var regex = "(?=" + tokens.join(")(?=") + ")";
+                        //console.log(regex);
+                        table = $('#{id}').DataTable();
+                        table.search(regex, true, false);
+                        table.draw();
```
Comment (alexjurkiewicz) [inline scoreboard/webutils.py:189]: Just checking -- is this necessary? I don't know datatables at all, but it feels unneeded

**Comment by alexjurkiewicz** [comment]
Added a few comments, please address those, otherwise LGTM 👍

File: scoreboard/webutils.py (hunk 3 above)
Comment (Kramin42) [reply scoreboard/webutils.py:189]: Yes, you are meant to .draw() afterwards, see https://datatables.net/reference/api/search()

**Pull request merged**